### PR TITLE
Add more checks for Universal SRS

### PR DESCRIPTION
**Pull request by Pratyush**
## Motivation

Forgot to push this commit in the last PR 
## Test Plan

(Write your test plan here)

## Related PRs

(Link your related PRs here)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,5 @@
 **/*.prover.*
 **/*.verifier.*
 **universal.srs.*.*
+*.usrs
```
Comment (howardwu) [inline .gitignore:8]: I see there are usrs files that are required, we should be pushing those on change and not ignoring them.

Comment (Pratyush) [reply .gitignore:8]: These are local to the repo, not to `.aleo/resources`, no?

Comment (howardwu) [reply .gitignore:8]: For 2^15 they're in the parameters folder

**Review by howardwu — APPROVED** [review]
LGTM

**Pull request merged**